utils/oscap_docker_python/oscap_docker_util.py

```diff
@@ -203,6 +203,17 @@ def _remove_mnt_dir(self, mnt_dir):
         if self.mnt_dir is None:
             os.rmdir(mnt_dir)
 
+    def _find_chroot_path(self, mnt_dir):
+        '''
+        Remember actual mounted fs in 'rootfs' for devicemapper
+        '''
+        rootfs_path = os.path.join(mnt_dir, 'rootfs')
+        if os.path.exists(rootfs_path):
+            chroot = rootfs_path
+        else:
+            chroot = mnt_dir
+        return chroot
```

Comment:
IMO it is, this is a C-ism that is unnecessary in python. Just return it, don't use a variable. Shorter is better IMO. You don't need to deallocate anything before you return which is why this pattern is even used in C.